tests/test_reporters.py

```diff
@@ -55,3 +56,32 @@ def test_vtk_reporter(tmpdir):
     simulation.step(2)
     assert os.path.isfile(tmpdir/"output_00000000.vtr")
     assert os.path.isfile(tmpdir/"output_00000001.vtr")
+
+
+@pytest.mark.parametrize("Flow", [TaylorGreenVortex2D, TaylorGreenVortex3D, 'DecayingTurbulence2D', 'DecayingTurbulence3D'])
+def test_EnergySpectrumReporter(tmpdir, Flow):
+    lattice = Lattice(D2Q9, device='cpu')
+    if Flow == TaylorGreenVortex3D or Flow is 'DecayingTurbulence3D':
+        lattice = Lattice(D3Q27, device='cpu')
+    if Flow is 'DecayingTurbulence2D' or Flow is 'DecayingTurbulence3D':
```

Comment:
I think these will be false always.  Use `==` instead.
`is` checks if two objects are identical (memory address)
`==` checks if their contents are identical

Reply:
The tests work fine with `is`. However `is` might be consistent within the code.

Reply:
maybe it works fine because it never uses the Decaying ??

Reply:
done

Reply:
It is. I checked locally if the "if condition" is `True` ;-)